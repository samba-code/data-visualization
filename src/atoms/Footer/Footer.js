import React from "react";
import styled from "styled-components";

import PropTypes from "prop-types";

const StyledFooter = styled.footer`
  display: flex;
  justify-content: center;
  width: 100%;
`;

const Footer = ({ children }) => <StyledFooter>{children}</StyledFooter>;

Footer.propTypes = {
    children: PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.node),
        PropTypes.node
    ]).isRequired,
};

export default Footer;