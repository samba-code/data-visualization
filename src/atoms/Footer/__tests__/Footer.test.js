import React from "react";
import { render } from "enzyme";

import Footer from "../Footer";

describe("Footer", () => {
  it("should match snapshot", () => {
    const wrapper = render(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
