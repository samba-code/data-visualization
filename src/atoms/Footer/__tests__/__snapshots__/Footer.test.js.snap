// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<footer
  class="Footer__StyledFooter-ge5quf-0 kftHFY"
>
  Footer
</footer>
`;
